### PR TITLE
Dynatrace-upgrade-perftest-01-testing

### PR DESCRIPTION
**Pull request by nickmoyni**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DTSPO-6187

### Change description ###

Adding oneagent classicfullstack code. Testing against perftest-01.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: apps/monitoring/perftest/01/dynatrace-operator.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: dynatrace-operator
+  namespace: monitoring
```
Comment (chrisorisawayi) [inline apps/monitoring/perftest/01/dynatrace-operator.yaml:6]: Is this being tested in monitoring namespace first before changing to dynatrace?

File: apps/monitoring/perftest/01/dynatrace-operator.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    spec:
+      chart: dynatrace-operator
+      version: 0.4.2
+      sourceRef:
+        name: dynatrace-operator
+        kind: HelmRepository
+        namespace: monitoring
+  valuesFrom:
+    - kind: Secret
+      name: oneagent-values
```
Comment (chrisorisawayi) [inline apps/monitoring/perftest/01/dynatrace-operator.yaml:20]: Same as base and should not be required.

Comment (chrisorisawayi) [inline apps/monitoring/perftest/01/dynatrace-operator.yaml:20]: Same as base and should not be required.

File: apps/monitoring/perftest/01/dynatrace-operator.yaml (hunk 1 above)
Comment (nickmoyni) [reply apps/monitoring/perftest/01/dynatrace-operator.yaml:6]: Ideally like to test under the current monitoring namespace. I did try to use dynatrace but this kept giving me an error.

File: apps/monitoring/perftest/01/dynatrace-operator.yaml (hunk 2 above)
Comment (nickmoyni) [reply apps/monitoring/perftest/01/dynatrace-operator.yaml:20]: Keeping this code, in the event i test with an alternative namespace.

Comment (chrisorisawayi) [reply apps/monitoring/perftest/01/dynatrace-operator.yaml:20]: In that case, the other elements apart from the namespace can be removed as these are already in the base.  Example

```
chart:
  spec:
    sourceRef:
      namespace: monitoring
```

Comment (nickmoyni) [reply apps/monitoring/perftest/01/dynatrace-operator.yaml]: Reduced the code as suggested.

File: apps/monitoring/perftest/01/dynatrace-operator.yaml (hunk 1 above)
Comment (nickmoyni) [reply apps/monitoring/perftest/01/dynatrace-operator.yaml:6]: I attempted to change namespace and this is where the error occurs.

**Review by chrisorisawayi — APPROVED** [review]

**Pull request merged**